### ISSUE TITLE
Closed groups with request-to-join feature

### ISSUE THREAD

**Issue by nicksellen** [yunity/karrot-frontend#566]
In #324 there was a mention of closed groups. This issue it to discuss that. I guess it needs a "request to join" feature too. Would probably also depend on having admins in practise (see #356).

A fancy version would be to enable an online conversation between the prospective new user and the admins of the group (would depend on #539 being implemented).

**Comment by tiltec** [comment]
Seems to be similar to #894, closing this in favor of the newer issue.
